src/Service/Configurations/RuntimeConfigValidator.cs

```diff
@@ -37,12 +37,13 @@ public class RuntimeConfigValidator : IConfigValidator
         // The claimType is invalid if there is a match found.
         private static readonly Regex _invalidClaimCharsRgx = new(_invalidClaimChars, RegexOptions.Compiled);
 
-        // Any reserved character is not allowed to be present in
-        // the rest/graphql custom path. Refer here: https://www.rfc-editor.org/rfc/rfc3986#page-12.
+        // "Reserved characters as defined in RFC3986 are not allowed to be present in the
+        // REST/GraphQL custom path because they are acceptable to be present in URIs.
```

Comment:
```suggestion
        // REST/GraphQL custom path because they are not acceptable to be present in URIs.
```
did you mean to say they are not acceptable?